demo/admin/src/products/categories/ProductCategoriesGrid.tsx

```diff
@@ -1,32 +1,35 @@
 import { gql, useApolloClient, useQuery } from "@apollo/client";
 import {
+    Button,
     CrudContextMenu,
     DataGridToolbar,
+    FillSpace,
     filterByFragment,
     type GridColDef,
+    StackLink,
     useBufferedRowCount,
     useDataGridRemote,
     usePersistentColumnState,
 } from "@comet/admin";
-import { useTheme } from "@mui/material";
-import { DataGridPro, type GridRenderCellParams, type GridRowOrderChangeParams, type GridToolbarProps } from "@mui/x-data-grid-pro";
-import { type ReactNode } from "react";
-import { useIntl } from "react-intl";
+import { Add as AddIcon, Edit as EditIcon } from "@comet/admin-icons";
+import { IconButton } from "@mui/material";
+import { DataGridPro, type GridRowOrderChangeParams, type GridSlotsComponent } from "@mui/x-data-grid-pro";
+import { FormattedMessage, FormattedNumber, useIntl } from "react-intl";
 
 import {
     type GQLCreateProductCategoryMutation,
     type GQLCreateProductCategoryMutationVariables,
     type GQLDeleteProductCategoryMutation,
     type GQLDeleteProductCategoryMutationVariables,
+    type GQLProductCategoriesGridFutureFragment,
     type GQLProductCategoriesGridQuery,
     type GQLProductCategoriesGridQueryVariables,
-    type GQLProductCategoryGridFutureFragment,
     type GQLUpdateProductCategoryPositionMutation,
     type GQLUpdateProductCategoryPositionMutationVariables,
 } from "./ProductCategoriesGrid.generated";
 
 const productCategoriesFragment = gql`
-    fragment ProductCategoryGridFuture on ProductCategory {
+    fragment ProductCategoriesGridFuture on ProductCategory {
```

Comment:
i think we can remove the Future postfix

Reply:
I agree: b191be4b40b741a6848b7b6a7e229396c2a4b5bd.